File: vjudge/26082024/g.py
n = int(input())
for _ in range(n):
	s = [*input()]
	new_s = ""
	anterior = ""
	if(s[0]!=s[1]):
		new_s += s[0]+s[0]
	else:
		new_s += s[0]
	for e in range(1, len(s)-2):
		if(s[e]!=s[e-1] and s[e]!=s[e+1]):
			new_s += s[e]+s[e]
		else:
			new_s += s[e]
	if(s[len(s)-2]!=s[len(s)-3]):
		new_s += s[len(s)-2]+s[len(s)-2]
	else:
		new_s += s[len(s)-2]
	print(new_s)

File: vjudge/26082024/g.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
	int n; cin>>n;
	for(int i = 0; i<n; i++){
		string s; cin>>s;
		string new_s = "";
		char anterior = ' ';
		if(s.size()==1) cout<<s<<s<<endl;
		else{
			if(s[0]!=s[1]){ new_s += s[0]; new_s += s[0];}
			else new_s += s[0];
			for(int e = 1; e<s.size()-1; e++){
				if(s[e]!=s[e-1] && s[e]!=s[e+1]){new_s += s[e]; new_s+=s[e];}
				else new_s += s[e];
			}
			if(s[s.size()-1]!=s[s.size()-2]){new_s += s[s.size()-1]; new_s+=s[s.size()-1];}
			else new_s += s[s.size()-1];
			cout<<new_s<<endl;
		}
	}
}
